AAInstrumentation/core.py

```diff
@@ -67,6 +70,119 @@ def generate_baseline(self):
             stderr=DEVNULL,
         )
 
+    def get_baseline_files(self) -> list[str]:
+        """Get the baseline files."""
+
+        files = [
+            Path(os.path.join(dirpath.removeprefix(str(self.initial_dir) + "/"), f))
+            for (dirpath, dirnames, filenames) in os.walk(
+                self.initial_dir.joinpath(self.benchmark)
+            )
+            for f in filenames
+        ]
+
+        return files
+
+    def measure_outputsize(self, file: Path) -> int:
+        cmd = [str(self.instr_path.joinpath("llvm-size")), str(file)]
+        p = run(cmd, stdout=PIPE, stderr=PIPE, text=True)
+        if p.stderr != "":
+            raise Exception("Error in measuring output size for " + str(file))
+        second_line = p.stdout.split("\n")[1]
+        line_list = second_line.split("\t")
+        # The other results are more info on where size is: text, data, bss
+        return int(line_list[0])
+
+    def assemble_file(self, file_name: Path, obj_file_name: Path):
+        """Assemble the file."""
+
+        cmd = [
+            str(self.instr_path.joinpath("llc")),
+            "-O2",
```

Comment:
Would it not make sense to not hardcode this?

Reply:
This is just for llc and I think this is what's being run for O3 and Oz, so I decided to hardcode it here. I guess I can also make it a class variable. What do you think?